### DISCUSSION TITLE
Functions to create new measures

### DISCUSSION THREAD

**Discussion by yemode2k** [atoti/atoti#754] [Q&A]
Is there any access to function menu to create new variables within atoti dashboard.

**Comment by tibdex** [comment]
Can you explain the use case more so that we can give an appropriate answer?

**Reply by yemode2k** [reply]
From the dashboard view I want to create new measures from existing measured for instance. A measure of total sales and total  cost and you wat to calculate total profit . Is there a way I can use functions within the dashboard and not going through the notebook

**Reply by tibdex** [reply]
You could do it by editing the `canUseCalculatedMeasures` [permission](https://activeviam.com/activeui/documentation/5.0.20/docs/api/types#permissions). To do that, if `session.port == 9090`, go to `http://localhost:9090/admin` (use `admin/admin` as credentials if prompted) and edit the `ui/organization_permissions` file to:

```json
{
  "canUseCalculatedMeasures": true
}
```

Then the `fx+` button should appear in the UI (refresh the browser tab if needed), click it and use the text area to define the calculated measure in MDX. Something like that:

![Screenshot 2023-02-14 at 17 08 57](https://user-images.githubusercontent.com/6574550/219411885-6bad6640-ecdf-446f-8bde-b6e2a977cb81.png)
